nest/src/config.ts

```diff
@@ -0,0 +1,23 @@
+export const config = () => ({
+    census: {
+        serviceID: process.env.CENSUS_SERVICE_ID,
+        environment: 'ps2',
+        worlds: process.env.CENSUS_SUBSCRIPTION_WORLDS.split(',') ?? ['all'],
+    },
+
+    rabbitmq: {
```

Comment:
Is this meant to be the admin queue?

Reply:
No, I wanted to make the aggregator a stand alone app, but I forgot that we use rabbitmq to mock instances. So this is the config for the microservice server that can be used to send events and request to the aggregator.